rpmlint/checks/Allowlisting.py

```diff
@@ -0,0 +1,428 @@
+# vim: sw=4 ts=4 sts=4 et :
+#############################################################################
+# Author        : Matthias Gerstner
+# Purpose       : reusable code for dealing with security allow lists
+#############################################################################
+
+import abc
+import hashlib
+import json
+import os
+import os.path
+import sys
+import traceback
+
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class DigestVerificationResult(object):
```

Comment:
inheritance of `object` is by default. There isn't a difference between `class Test:`  and `class Test(object):`